modules/programs/waybar.nix

```diff
@@ -0,0 +1,378 @@
+{ config, lib, pkgs, ... }:
+
```

Comment:
Can add
```suggestion
with lib;
```
here to avoid having to prefix so many function calls by `lib.`.

Reply:
Thanks